src/ResourceState.cpp

```diff
@@ -159,9 +159,9 @@ namespace D3D12TranslationLayer
             return GetCommandListTypeMask(0);
         }
         UINT TypeMask = 0;
-        for (UINT i = 0; i < m_spExclusiveState.size(); ++i)
+        for (size_t i = 0; i < m_spExclusiveState.size(); ++i)

```

Comment:
This one should cast the size down, since it casts the iterator down inside the loop.

Reply:
so it does, thanks :)